harness/determined/exec/prep_container.py

```diff
@@ -52,15 +59,19 @@ def trial_prep(sess: api.Session, info: det.ClusterInfo) -> None:
         )
         raise
 
-    tgz = base64.b64decode(model_def_resp.to_json()["b64Tgz"])
+    b64_tgz = context_directory_resp.to_json()["b64Tgz"]
```

Comment:
nit: It's not a bug or anything, but this seems more complicated than
```python
b64_tgz = context_directory_resp.b64Tgz
```
for the same functionality.